### DISCUSSION TITLE
Is this HW3, v10.2 issues?

### DISCUSSION THREAD

**Discussion by Oswalds99** [sanni/cartreader#616] [Need help]
Hallo

I tried pressing Read Rom on SEGA Genesis in HW3, v10.2 But returns to the first screen.
Also, noise appears on the Reading screen.
Sometimes I see garbled characters are occasionally seen in other versions than v10.2, but the SEGA error is not present in other versions.
Is this HW3, v10.2 issues?
https://user-images.githubusercontent.com/111804208/200463538-5ab29b73-19cb-436d-b2c2-ddf638994304.mov
https://user-images.githubusercontent.com/111804208/200463546-b707e33b-87e6-457c-bf74-53e41f2159f3.mov

**Comment by Cephiros** [comment]
I have seen similar with n64, and another system, possibly genesis-md on hardware5 with 10.2, but it doesnt seem to effect anything meaningful. I suspect it may be memory, or lack there-of related. Aside from aesthetics, does yours effect the dumps in any way? 
![20221022_200649](https://user-images.githubusercontent.com/9114282/200477427-90f406cb-2e84-4be5-91f1-796a6f946c21.jpg)

**Reply by Oswalds99** [reply]
It has no effect on dumps.
MD cannot dump and returns to the first screen.
But problem was solved in v11.2, which was made by sanni.
Thanks for the reply.

**Comment by kamalawala** [comment]
Noise is related to #388. It may look different due to the OLED screen of the HW3; I noticed that the code flips the screen 180° for the LCD of HW4/5, which may not be the case for HW3 (can't look at the code right now). Try disabling modules as it says and see if it helps with the Mega Drive menu. I don't have a guess as to the cause of that one and I can't test, as I don't have a HW3.

**Reply by Oswalds99** [reply]
Thanks for the reply.
The problem was solved in v11.2, which was made by sanni.

**Comment by sanni** [comment]
> Is this HW3, v10.2 issues?

Can you try out the current [V11.2](https://github.com/sanni/cartreader/archive/refs/heads/master.zip) code?

**Reply by Oswalds99** [reply]
No problems seen with both in V11.2
Thank you for awesome work
